### ISSUE TITLE
Remove hardcoded kernel version in ROCm. 

### ISSUE THREAD

**Issue by sameershende** [RadeonOpenCompute/ROCm#1221]
While compiling ROCm for Ubuntu 20.04 I see a couple of places where the kernel version has been hardcoded. It may be better for a script to detect the kernel version and inject it in these places. Specifically, 3.6.0 is assumed in:

1. ROCK-Kernel-Driver/include/config/kernel.release 
2. ROCK-Kernel-Driver/include/config/auto.conf.cmd

files.

**Comment by ROCmSupport** [comment]
Thanks @sameershende for reaching out.
I will check and get back asap on this issue.
Thank you.

**Comment by kentrussell** [comment]
These are generated during the package creation for each release. Is there a problem with the values? he 3.6.0 references the ROCm 3.6.0 release, not kernel 3.6. If there is something that is going wrong with that value, we can bring it up with the Packaging team

**Comment by ROCmSupport** [comment]
Hi @sameershende 
Request you to respond asap so that we can do something.
Thank you.

**Comment by sameershende** [comment]
I can't find kernel.release and auto.conf.cmd files in the current repo, so it is difficult to check the values with the current release. I just checked my notes and the value that was hardcoded was not 3.6.0 - it was 5.6.0. Clearly, it couldn't have been a ROCm release id. It was an OS kernel version number that was just hardcoded in these two files (auto.conf.cmd and kernel.release). For Ubuntu 20.04, I had to modify it manually to 5.4.0-42-generic. And the variable that was assigned this value was KERNEL_VERSION. If a high level script to build ROCm on multiple platforms (including ppc64le) is created, it would have to detect this version from the output of uname and inject it into these files. But since these files are not used for the build,  perhaps this issue has been resolved.
